### DISCUSSION TITLE
Welcome to fun-repo Discussions!

### DISCUSSION THREAD

**Discussion by Naga-Manohar-Y** [Manohar-Git-Cloud/fun-repo#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EpicSpaceFan** [comment]
Hey, I'm a new GitHub person and I made an interesting project to view Earth using NASA's API called "EPIC".
Here's the link: https://github.com/EpicSpaceFan/EarthViewer
